code/modules/mob/mob.dm

```diff
@@ -663,6 +663,8 @@ var/list/slot_equipment_priority = list( \
 		return 0
 
 	for(var/slot in slot_equipment_priority)
+		if(!is_holding_item(W))
```

Comment:
Does this have a reason to be inside the for loop?

Reply:
Yes. Because of the do_after, you could throw the mask (or any future items that have a delayed equip), and it would warp straight into your pocket